### ISSUE TITLE
For node.js add module.exports

### ISSUE THREAD

**Issue by nowshad-sust** [aishek/axios-rate-limit#3]
So, the below code block doesn't work for nodejs apps, may be you targeted the frontend env.
```
exports.__esModule = true
exports.name = 'axiosRateLimit'
exports.default = axiosRateLimit
```

Adding one more line for the nodejs type will do the job:
```
module.exports = axiosRateLimit
```

**Comment by aishek** [comment]
Oh, thank for the issue! I just published https://www.npmjs.com/package/axios-rate-limit/v/1.0.0 to fix it.

Could you confirm that the issue is fixed in version `1.0.0`?

**Comment by nowshad-sust** [comment]
@aishek thanks for the quick update 😃
